src/learner.cc

```diff
@@ -857,6 +857,25 @@ class LearnerConfiguration : public Learner {
       mparam_.num_target = n_targets;
     }
   }
+
+  void InitEstimation(MetaInfo const& info, linalg::Tensor<float, 1>* base_score) {
```

Comment:
What happens if we just calculate the gradient using individual workers? Is the gradient still the same? If so, we can just let them calculate.

Reply:
Since we don't have labels in non-0 workers, they won't be able to calculate the gradient.